layouts/partials/navbar-version-selector.html

```diff
@@ -3,8 +3,8 @@
 </a>
 <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownMenuLink">
 	{{ $p := . }}
-	<a class="dropdown-item" href="/releases">Release Information</a>
+	<a class="dropdown-item" href="{{ "releases" | relLangURL }}">{{ i18n "release_information_navbar" . }}</a>
```

Comment:
Hi @a-mccarthy .
What is the reason for removing the forward slash (/) in the `href`?

Reply:
@kbhawkey great question! I just double checked, the hugo function [relLangURL](https://gohugo.io/functions/rellangurl/) should put the `/` in the output, so i dont think its necessary anymore